doc/administrator_guide.md

```diff
@@ -94,6 +94,28 @@ Select the account key for use with Google Cloud API (which kcidb uses):
 Clone kcidb repository, checkout the revision you want to deploy, and install
 kcidb as described in [README.md](README.md).
 
+### Additional Dependencies
+
+While deploying you may also need to install psql. psql is a command-line tool for interacting with the PostgreSQL database. It is used to manage and manipulate data stored in a PostgreSQL database, as well as to administer and configure the database itself. 
+
+Update the package lists:
+    
+    sudo apt-get update
+
+Install the postgresql-client-common package: 
+
+    apt-get install postgresql-client-common
+
+Install the postgresql-client package, which includes the psql command:
+   
+    apt-get install postgresql-client
+
+Verfiy the psql installation: 
+   
+    psql --version
+
+Once the installation is complete, you can start using psql to interact with PostgreSQL databases. You can refer to the documentation here https://www.postgresql.org/docs/current/installation.html
```

Comment:
We don't need this sentence, as we don't expect the users to interact with the tool by themselves, only through the `cloud` script. Moreover, if they intend to use the tool themselves, they would already know quite a lot about it.